modules/creating-machines-bare-metal.adoc

```diff
@@ -23,7 +23,10 @@ You can configure {op-system} during ISO and PXE installations by using the foll
 * Ignition configs: {product-title} Ignition config files (`*.ign`) are specific to the type of node you are installing. You pass the location of a bootstrap, control plane, or compute node Ignition config file during the {op-system} installation so that it takes effect on first boot. In special cases, you can create a separate, limited Ignition config to pass to the live system. That Ignition config could do a certain set of tasks, such as reporting success to a provisioning system after completing installation. This special Ignition config is consumed by the `coreos-installer` to be applied on first boot of the installed system. Do not provide the standard control plane and compute node Ignition configs to the live ISO directly.
 
 * `coreos-installer`: You can boot the live ISO installer to a shell prompt, which allows you to prepare the permanent system in a variety of ways before first boot. In particular, you can run the `coreos-installer` command to identify various artifacts to include, work with disk partitions, and set up networking. In some cases, you can configure features on the live system and copy them to the installed system.
++
```

Comment:
One thought to consider is that ISG suggests using "later" to refer to software versions.

https://www.ibm.com/docs/en/ibm-style?topic=word-usage#l

So maybe this could be changed to "As of version `0.17.0-3`, `coreos-installer` requires {op-system-base} 9 or later to run the program.